stats/stats.go

```diff
@@ -86,7 +95,7 @@ func (t MetricType) MarshalJSON() ([]byte, error) {
 }
 
 // UnmarshalJSON deserializes a MetricType from a string representation.
```

Comment:
```suggestion
// UnmarshalText deserializes a MetricType from a string representation.
```

Weird that `golangci-lint` missed this... :-/

Reply:
Resolved by https://github.com/loadimpact/k6/pull/1768/commits/970122cd9733bc25a1630fc2072ba1257a2e1407